addons/mass_mailing/views/themes_templates.xml

```diff
@@ -7,7 +7,9 @@
             <p><br/></p>
             <p>
                 <br/>
-                <a href="/unsubscribe_from_list">Unsubscribe</a>
+                <span class="unsubscribe_link">
```

Comment:
Well, bof, this is adding dom just for backend display. Can't we inline the style somehow ?